dataset_generation/hi_wikisplit/readme.md

```diff
@@ -0,0 +1,16 @@
+# Generation of a dataset from wikipedia revision history
+
+The method outlined in the paper [Learning to Split and Rephrase From Wikipedia Edit History](https://arxiv.org/abs/1808.09468) was followed.
+
+## Execution:
+
+Download and unzip the title dump file from [https://dumps.wikimedia.org/other/pagetitles/20200503/](https://dumps.wikimedia.org/other/pagetitles/20200503/)
+
+Run Scraper.py with the path to the title dump file, followed by Dataset.py
```

Comment:
please change this too according to the file renames. Also, can we sort of have a set of instructions here that anyone can use for a new language - it is not very different from what is already present this file just in pointwise steps.

Reply:
yes, that makes sense.